index.js

```diff
@@ -0,0 +1,228 @@
+    #scoreContainer = this.find('#snake-score');
+    #score = Number(this.#scoreContainer.firstElementChild.innerHTML)
+
+    constructor({boxSize, gridCount}) {
+        super({
+            boxSize,
+            gridCount,
+            gridCellCssClass: Snake.snakeCellCssClass,
+            gridContainerSelector: Snake.gridContainerSelector,
+        });
+        this.direction = D.LEFT;
+        this.#init(); 
+
+    }
+
+    #init() {
+        this.#startBtn.addEventListener('click', () => { 
+            this.#start();
+        })
+
+        this.#endBtn.addEventListener('click', () => { 
+            this.#endGame();
+        })
+
+        document.addEventListener('keydown', (event) => {
+            this.#updateDirection(event);
+        })
+    }
+
+    #start() {
+        let middleCell = Math.floor(this.gridCount) /2;
+        this.#snake = this.#buildSnake(middleCell, middleCell, this.#size) ; 
+        this.#speed = +this.#form.speed.value; 
+         
+
+        this.#startBtn.style.display = 'none'; 
+        this.#endBtn.style.display = 'block'; 
+        this.#messageContainer.innerHTML = 'Welcome to Snake !' 
+
+        this.#process = setInterval(() => {
+
+            let { cell, row } =  this.#snake[0]; 
+            let snakePatrToShift = null; 
+
+            switch(this.direction) {
+                case D.LEFT : {
+                    snakePatrToShift = {
+                        cell: cell - 1,
+                        row
+                    }
+                }; break;
+                case D.RIGHT: {
+                    snakePatrToShift = {
+                        cell: cell + 1,
+                        row
+                    }
+                }; break;
+                case D.UP: {
+                    snakePatrToShift = {
+                        cell,
+                        row: row - 1
+                    }
+                }; break;
+                case D.DOWN: {
+                    snakePatrToShift = {
+                        cell,
+                        row: row + 1
+                    }
+                }; break;
+            };
+            snakePatrToShift = this.#noWallMode(snakePatrToShift) 
+            this.#snake.unshift(snakePatrToShift);
+            this.#clear();
+            this.#update(); 
+        },this.#speed)
+        this.#generatorFood();
+         
+ 
+    } 
+    #noWallMode({cell, row}) {
+        if (cell === -1  ) {
+            cell = cell + this.gridCount;   
+        }
+        else if (cell === 12 ) {
+            cell = 0;
+        }
+        if (row === -1  ) {
+            row = row + this.gridCount;   
+        }
+        else if (row === 12 ) {
+            row = 0;
+        } 
+        return {cell, row}
+    }
+
+    #removeАppleрEnd({pointColumn, pointRow}) {
+        let imgBlok = this.find(`[data-cell="${pointColumn}"][data-row="${pointRow}"]`, this.gridContainer);
+        imgBlok.removeChild(imgBlok.firstElementChild);
+    }
```

Comment:
a lot of typos here (`removeAppleEnd`, `imgBlock`). 
Also you can name the method just `removeApple`